config/locales/en.yml

```diff
@@ -105,7 +105,7 @@ en:
         address_telephone: "Contact telephone number"
         phone_call_scheduled_at: "Select your preferred day and time for a callback"
       teacher_training_adviser_steps_overseas_country:
-        country_id: "Which country do you live in?"
+        country_id: Are you a non-UK citizen? Please select your country.
```

Comment:
I think there might be a bit of confusion around this step and the preceding one if the person is a UK citizen that is currently living overseas - they hit a bit of a dead-end?

> Where do you live?

Overseas

> Are you a non-UK citizen? Please select your country.

No?